test/regression.uts

```diff
@@ -6879,3 +6879,66 @@ x = f.i2repr(mp, {'*', '+', 'bit 2'})
 assert(re.match(r'^.*Star \(\*\).*$', x) is not None)
 assert(re.match(r'^.*Plus \(\+\).*$', x) is not None)
 assert(re.match(r'^.*bit 2.*$', x) is not None)
+
+###########################################################################################################
```

Comment:
The reason for that is that I'm able to test my code and not the code of `socket.inet_ntop()` (also that is the reason that I extracted a separate function)
Do you see a better way to achieve this?